spec/controllers/answer_tags_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+describe AnswerTagsController do
+
+  #factory objects required for "action_allowed" test cases
+  let(:instructor) { build(:instructor, id: 1) }
```

Comment:
Is this line needed? We are not using `instructor` anywhere else in the code.

Reply:
Yeah actually not needed will remove this in next iteration.